usersession/userd/userd.go

```diff
@@ -77,18 +77,24 @@ func (ud *Userd) Init() error {
 		// at the object level and the actual well-known object name
 		// becoming available on the bus
 		xml := "<node>" + iface.IntrospectionData() + introspect.IntrospectDataString + "</node>"
-		ud.conn.Export(iface, iface.BasePath(), iface.Name())
-		ud.conn.Export(introspect.Introspectable(xml), iface.BasePath(), "org.freedesktop.DBus.Introspectable")
+		ud.conn.Export(iface, iface.ObjectPath(), iface.Interface())
+		ud.conn.Export(introspect.Introspectable(xml), iface.ObjectPath(), "org.freedesktop.DBus.Introspectable")
 
+	}
+
+	for _, name := range []string{
+		"io.snapcraft.Launcher",
+		"io.snapcraft.Settings",
+	} {
```

Comment:
should these names be put into a variable to make it clear that this list is specifically meant to be static and not changing over time?

Reply:
Good idea.  I've moved these definitions to a global, and added a comment about not needing to extend the list to add new interfaces.